File: 1-TwoSum/1-TwoSum.java
// Last updated: 6/26/2025, 10:42:58 PM
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[]result=new int[2];
        for(int i=0;i<nums.length;i++){
            for(int j=i+1;j<nums.length;j++){
                if(nums[i]+nums[j]==target){
                    result[0]=i;
                    result[1]=j;
                }
            }
        }
        return result;
    }
}

File: 1-TwoSum/1-TwoSum.cpp
// Last updated: 6/26/2025, 10:42:57 PM
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> vec;
        for (int i = 0; i < nums.size(); i++) {
            for (int j = i + 1; j < nums.size(); j++) {
                if (nums[i] + nums[j] == target) {
                    vec.push_back(i);
                    vec.push_back(j);
                    return vec;
                }
            }
        }
        return vec;
    }
};